packages/exhibitor/extra/start_exhibitor.py

```diff
@@ -49,6 +54,7 @@ def invoke_detect_ip():
     '-Djava.util.prefs.userRoot=/var/lib/dcos/exhibitor/',
     '-Duser.home=/var/lib/dcos/exhibitor/',
     '-Duser.dir=/var/lib/dcos/exhibitor/',
+    '-Djna.tmpdir=%s' % jna_tmpdir,
```

Comment:
`SERVER_JVMFLAGS` is zookeeper-specific. I think we should keep that in the service file as the reader can immediately grep for that string and understand that this gets read by the zookeeper start script.

`-Djna.tmpdir` is specific to exhibitor.

It seems clear that we need to specify both values separately. Whether we need to extract the path into a separate variable is another matter. I don't think it would add much. I'm a "refactor on third use" kind of guy.